commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -30,6 +30,7 @@ def parse_subject(text):
 class ConventionalCommitsCz(BaseCommitizen):
     bump_pattern = defaults.bump_pattern
     bump_map = defaults.bump_map
+    bump_map_major_version_zero = defaults.bump_map_major_version_zero
```

Comment:
If this is not actually a default for every cz, should we move it out of `defaults`? I've been thinking for moving these kind of stuff from defaults for a long time but not yet have a chance to do so 🤦‍♂️